Framework/Algorithms/test/CalculateMultipleScatteringTest.h

```diff
@@ -103,25 +112,29 @@ class CalculateMultipleScatteringTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_NOTHING(alg->setProperty("InputWorkspace", inputWorkspace));
     const int NSCATTERINGS = 2;
     TS_ASSERT_THROWS_NOTHING(alg->setProperty("NumberScatterings", NSCATTERINGS));
-    TS_ASSERT_THROWS_NOTHING(alg->setProperty("NeutronPathsSingle", 10000));
-    TS_ASSERT_THROWS_NOTHING(alg->setProperty("NeutronPathsMultiple", 10000));
+    TS_ASSERT_THROWS_NOTHING(alg->setProperty("NeutronPathsSingle", 100000));
```

Comment:
I checked this in release and debug mode and it increases the test time from 10->20s in Debug and 1 to 2 in Release

For people developing on the algorithm on MSVC Debug in the future, do we need the higher number of paths?

Reply:
The change I'd made to the way the random number seed is generated:

```MersenneTwister rng(seed + specNo);```

meant that this test wasn't passing any more. I concluded that I'd simply got lucky before in that the Monte Carlo simulation wasn't properly converged but the default seed happened to give an answer v close to the theoretical result. In order to get it to pass I had to increase the number of scenarios so the simulation was more fully converged. So I think this is unavoidable unfortunately - I could reduce the tolerance of the TS_ASSERT_DELTA later on I guess but the extra run time didn't seem to bad?

Reply:
On balance I'd rather have the longer runtime and increased accuracy rather than a fast inaccurate test :+1: